### PR TITLE
Setup md

### PR DESCRIPTION
**Pull request by ledm**
This is a continuation of the now merged setup.md PR #45 #46 

# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [ ] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes {Link to the corresponding issue}

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -53,27 +51,35 @@ The benefit of using a compute cluster with an ESGF node is that the
...
-If neccesairy, data can be downloaded using the 
+While some data will be provided by your tutors, there is simply too much data 
+for your tutors to make it all available directly.
+
+If you do not have access to an ESGF node, data can be downloaded using the 
 [synda tool](https://prodiguer.github.io/synda/index.html).
- 
+
+Depending on which machine you plan to use, you will need to work through one set of the following instructions:
+- [CEDA-Jasmin (UK)](#CEDA-Jasmin)
```
Comment (ledm) [inline setup.md:73]: These links still don't work!

Comment (Peter9192) [reply setup.md:73]: Don't use capitals: `#ceda-jasmin` works for me, just like `#dkrz` and `#your-local-machine`.

**Comment by ledm** [comment]
This PR is failry limited, as most of the changes were merged in another PR. 

There are just a few minor changes suggested here. It's not the end of the world if they don't make it in, there's nothing really groundbreaking here.

**Comment by JaroCamphuijsen** [comment]
@ledm shall we then close this pr and remove the branch?

**Comment by ledm** [comment]
There are still some gaps in the setup.md, we might as well make these changes today, while we can.

File: setup.md (hunk 1 above)
Comment (sverhoeven) [reply setup.md]: fixed

**Pull request merged**